script/URMOZDeployInput.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: MIT
+// slither-disable-start reentrancy-benign
+
+pragma solidity 0.8.30;
+
+contract URMOZDeployInput {
```

Comment:
nit: i might suggest noting somewhere that these input values are using an arbitrary OZ governor (in this case, Obol's) that we chose for convenience in helping us test

^ is that true?

Reply:
This is true. Will add that as a comment